lib/src/Base/Type/openturns/SquareMatrix.hxx

```diff
@@ -139,6 +139,9 @@ public:
   ComplexCollection computeEigenValues(const Bool keepIntact = true);
   ComplexCollection computeEV(SquareComplexMatrix & vOut,
                               const Bool keepIntact = true);
+  /** Compute an approximation of the largest eigenValue module using power iterations */
+  Scalar computeApproximateLargestEigenValueModule(const UnsignedInteger maximumIterations = ResourceMap::GetAsUnsignedInteger("Matrix-LargestEigenValueIterations"),
+                                                   const Scalar epsilon = ResourceMap::GetAsScalar("Matrix-LargestEigenValueRelativeError")) const;
```

Comment:
missing `virtual`

Reply:
Why do you want to make this method virtual? None of the other methods are virtual...

Reply:
Because you need to change method content for children classes